src/index.html

```diff
@@ -6,14 +6,483 @@
+              class="contact-form__input"
+              required
+            />
+            <textarea
+              type="text"
+              name="message"
+              placeholder="Your message"
+              class="contact-form__input contact-form__input--message"
+              required
+            ></textarea>
+            <button
+              type="submit"
+              class="button contact-form__button"
+            >
+              Send
+            </button>
+          </form>
+
+          <address
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="contacts__title">Contact us</h2>
+            <div class="contacts__wrapper">
+              <label class="contacts__item">
+                call us
+                <a
+                  href="tel:+654 321 987"
+                  class="contacts__link"
+                >
+                  654 321 987
+                </a>
+              </label>
+
+              <label class="contacts__item">
+                visit us
+                <a
+                  href="https://www.google.com/maps/@42.169758,-88.0347239,12z?entry=ttu"
+                  target="_blank"
+                  class="contacts__link"
+                >
+                  2905 West Drive, Buffalo Grove
+                </a>
+              </label>
+
+              <label class="contacts__item">
+                our socials
+                <div class="contacts__social-wrapper">
+                  <a
+                    href="http://facebook.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--facebook"
+                  ></a>
+
+                  <a
+                    href="http://twitter.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--twitter"
+                  ></a>
+
+                  <a
+                    href="http://instagram.com"
+                    target="_blank"
+                    class="contacts__social contacts__social--instagram"
+                  ></a>
+                </div>
+              </label>
+            </div>
+          </address>
+        </div>
+        <div class="footer__bottom">
+          <a href="#">
+            <img
+              src="./images/logo/dia-logo.svg"
+              alt="logo"
+              class="footer__logo"
+            />
+          </a>
+          <nav class="nav">
+            <ul class="nav__list nav__list--footer">
+              <li class="nav__item">
+                <a href="#about">about us</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#services">services</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#testimonials">testimonials</a>
+              </li>
+
+              <li class="nav__item">
+                <a href="#contacts">contact us</a>
+              </li>
+            </ul>
```

Comment:
These links are also not being highlighted on hover [Referece](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Hightlight%20clickable%20element%20with%20cursor%3A%20pointer%20and%20some%20%3Ahover%20styles)